packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/portabilities/telecom-telephony-alias-portability-portabilities.controller.js

```diff
@@ -4,79 +4,112 @@ import get from 'lodash/get';
 import map from 'lodash/map';
 import set from 'lodash/set';
 
-angular.module('managerApp').controller('TelecomTelephonyAliasPortabilitiesCtrl', function TelecomTelephonyAliasPortabilitiesCtrl($translate, $stateParams, $q, OvhApiTelephony, TucToast) {
-  const self = this;
+function groupPortaByNumbers(portabilities) {
+  let numbers = [];
+  forEach(portabilities, (porta) => {
+    numbers = porta.numbersList.map(number => ({
+      number,
+      portability: porta,
+      lastStepDone: find(porta.steps.slice().reverse(), { status: 'done' }),
+    }));
+  });
+  return numbers;
+}
 
-  self.loading = {
-    cancel: false,
-  };
+angular.module('managerApp').controller('TelecomTelephonyAliasPortabilitiesCtrl', class TelecomTelephonyAliasPortabilitiesCtrl {
```

Comment:
Hmm ... I think the above function should either be in another file (a `service` maybe) or in the class